src/utils.ts

```diff
@@ -13,7 +13,7 @@ export const VAULTS = {
   TestNet: {
     // default decimals are 6 -> which scales to 1e6 (1e6 microAlgos = 1 Algo)
     algo: {
-      vaultId: 88664016,
+      vaultId: 100233411,
```

Comment:
We should remove goBTC/goETH/dAlgo for this update, because there are not going to be any ASA vaults

Reply:
@AwolDes 
Is it that there won't be any asa vault or the previous asa vaults will still be available on the frontend and just that a new asa vault will not be available on the frontend?

Reply:
This version of the SDK will not support any ASA vaults, so we should remove the reference